embedder.py

```diff
@@ -0,0 +1,66 @@
+"""
+Created on 2022 Jan 28 14:09 
+
+@author: keller
+"""
+import abc
+
+import torch
+import ruclip
+import numpy as np
+
+from PIL import Image
+
+from numbers import Number
+from typing import List
+
+class Embedder(abc.ABC):
+    @abc.abstractmethod
+    def encode_text(self, text):
+        pass
+
+    @abc.abstractmethod
+    def encode_imgs(self, imgs):
+        pass
+
+    def cos(self, text_emb: np.ndarray, img_emb: np.ndarray) -> Number:
+        """
+        Returns cos similarity between text_emb and img_emb
```

Comment:
Unnecessary mention about `text_emb` and `img_emb`, it compares just 2 embeddings

Reply:
Ok